### PR TITLE
Add descriptive files for Rancher UI to show for this chart

### PR DESCRIPTION
**Pull request by ereslibre**
Fixes https://github.com/kubewarden/helm-charts/issues/8

### CODE REVIEW

**Comment by ereslibre** [comment]
This PR clashes with another one that has to be opened to bump the policy-server version when it is published in the registry.

**Comment by ereslibre** [comment]
Rebased on top of latest helm charts.

File: charts/kubewarden-controller/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+[pod-privileged](https://github.com/kubewarden/pod-privileged-policy)
+policy:
+
+```yaml
+apiVersion: policies.kubewarden.io/v1alpha1
+kind: ClusterAdmissionPolicy
+metadata:
+  name: privileged-pods
+spec:
+  module: registry://ghcr.io/kubewarden/policies/pod-privileged:v0.1.0
```
Comment (flavio) [inline charts/kubewarden-controller/README.md:54]: Can you update that to be `v0.1.5`? I fear that is a really old version that would not work.

Also, I think you have copied that from somewhere else, can you update the original document too?

File: charts/kubewarden-controller/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  resources:
+  - pods
+  operations:
+  - CREATE
+  - UPDATE
+  settings:
+    trusted_users:
+    - alice
```
Comment (flavio) [inline charts/kubewarden-controller/README.md:62]: Oh my, this is a really old example, it's using the old trusted_users thing that was removed.

Can you use the contents of the quick start maybe?

**Review by flavio — CHANGES_REQUESTED** [review]
I didn't try that into the Rancher UI, but it looks good to me. I left just some notes

File: charts/kubewarden-controller/README.md (hunk 1 above)
Comment (ereslibre) [reply charts/kubewarden-controller/README.md:54]: Yes, I will come back to this. I have been fixing references to policies so we are on latest versions given some SDK changes, just to be on the safe side (waPC vs WASI, etc)

**Comment by ereslibre** [comment]
Merging after addressing all comments.

**Pull request merged**